index.js

```diff
@@ -8,13 +8,22 @@ const ApiDocumentation = require('./lib/ApiDocumentation')
 const BodyParser = require('./lib/BodyParser')
 const IriTemplate = require('./lib/IriTemplate')
 const Router = require('express').Router
-const SparqlView = require('./lib/SparqlView')
+const createRegistry = require('./lib/loaders')
+const cf = require('clownface')
+const SparqlHttp = require('sparql-http-client')
 
 function middleware (apiPath, api, options) {
   options = options || {}
 
+  const loaders = createRegistry(options)
   const graph = options.graph || api
 
+  const client = new SparqlHttp({
+    endpointUrl: options.sparqlEndpointQueryUrl || options.sparqlEndpointUrl,
+    updateUrl: options.sparqlEndpointUpdateUrl,
+    fetch: rdfFetch
+  })
```

Comment:
I moved the SPARQL client to the very top.

Reply:
Makes sense. The client should be available not only to the SPARQL code, but also to the plain JS middlewares.